react/features/base/modal/components/JitsiKeyboardAvoidingView.js

```diff
@@ -54,8 +61,10 @@ const JitsiKeyboardAvoidingView = (
     const tabNavigatorPadding
         = hasTabNavigator ? headerHeight : 0;
     const noNotchDevicePadding = bottomPadding || 10;
-    const iosVerticalOffset = headerHeight + noNotchDevicePadding + tabNavigatorPadding;
-    const androidVerticalOffset = headerHeight;
+    const iosVerticalOffset
+        = headerHeight + noNotchDevicePadding + tabNavigatorPadding;
+    const androidVerticalOffset = isInputCoveredByAndroidKeyboard
+        ? headerHeight + StatusBar.currentHeight : headerHeight;
```

Comment:
What's the value of `StatusBar.currentHeight` ? IIRC we hide it while in a conference so it should be zero.

Reply:
![Webp net-resizeimage](https://user-images.githubusercontent.com/26413496/145218055-32a01714-ee71-4488-96e4-51f19a07f454.jpg)

Reply:
![image](https://user-images.githubusercontent.com/26413496/145233797-c309dbf3-37e6-456b-a60c-7e167fa06147.png)
This is a screenshot from a physical Pixel 6, taken by Damian.

Reply:
Alrighty!

Reply:
It's really weird that on iOS, the status bar height is undefined and on Android it's 49.